glasbey.py

```diff
@@ -1,29 +1,36 @@
 #!/usr/bin/env python
 # encoding: utf-8
-
-import sys
+import os, sys
```

Comment:
Please re-add empty line before and put one import per line.

Reply:
Done.